monorepo-migration/generate-replace-message-expressions.py

```diff
@@ -57,6 +57,37 @@ def main():
                 )
                 regexes.append(regex)
 
+    # fix up wrong replacements from previous migrations
```

Comment:
This is an impressive list!

Reply:
+1 Thank you for finding and fixing these edge cases we missed in the previous migrations!